doc/news/changes/incompatibilities/20201129Bangerth

```diff
@@ -0,0 +1,8 @@
+Removed: The file `affine_constraints.h` had a class called
+`IsBlockMatrix` that, however, also allowed to check whether its
+template argument is in fact a block sparsity pattern. The class was
+likely never intended to be in the public interface, and has been
+moved to internal::AffineConstraints::IsBlockMatrix and now also
+internal::AffineConstraints::IsBlockSparsityPattern.
```

Comment:
`now also`? The sentence structure does not seem to be right!?

Reply:
Well, it has been "moved to X and also Y" because it was split into two classes in #11244. I think this is grammatically and semantically correct.